controllers/topology/configmap.go

```diff
@@ -28,6 +32,11 @@ func renderConfigMap(
 	}
 
 	for nodeName, nodeTopo := range clabernetesConfigs {
+		// always initialize the tunnels keys in the configmap, this way we don't have to have any
+		// special handling for no tunnels and things always look consistent; we'll override this
+		// down below if the node has tunnels of course!
+		configMap.Data[fmt.Sprintf("%s-tunnels", nodeName)] = ""
```

Comment:
so the empty `%s-tunnels` element won't cause attempt to create tunnels down the road? I presume no, but just wanted to check if the function that reacts to the empty tunnels files appropriately.

Reply:
Yeah just will unmarshal an empty slice of tunnels then we iterate over nothing. Tested quickly and seemed to do the trick :)